client/app/components/app-header/AppHeader.jsx

```diff
@@ -0,0 +1,252 @@
+import { $location, $route } from '@/services/ng';
+import navigateTo from '@/services/navigateTo';
+import { Dashboard } from '@/services/dashboard';
+import { Query } from '@/services/query';
+import frontendVersion from '@/version.json';
+import logoUrl from '@/assets/images/redash_icon_small.png';
+
+import './AppHeader.less';
+
+function onSearch(q) {
+  $location.path('/queries').search({ q });
+  $route.reload();
+}
+
+function Desktop() {
+  return (
+    <div className="app-header" data-platform="desktop">
+      <div>
+        <Menu mode="horizontal" selectable={false}>
+          <Menu.Item key="dashboards" className="dropdown-menu-item">
+            <Button onClick={() => navigateTo('dashboards')}>
+              Dashboards
+            </Button>
+            <DropdownFavorites fetch={Dashboard.favorites} urlTemplate="dashboard/${slug}" />
+          </Menu.Item>
+          <Menu.Item key="queries" className="dropdown-menu-item">
+            <Button onClick={() => navigateTo('queries')}>
+              Queries
+            </Button>
+            <DropdownFavorites fetch={Query.favorites} urlTemplate="queries/${id}" />
+          </Menu.Item>
+          <Menu.Item key="alerts">
+            <Button onClick={() => navigateTo('alerts')}>
+              Alerts
+            </Button>
+          </Menu.Item>
+        </Menu>
+        <Dropdown
+          trigger={['click']}
+          overlay={(
+            <Menu>
+              {currentUser.hasPermission('create_query') && (
+                <Menu.Item key="new-query">
+                  <a href="queries/new">New Query</a>
+                </Menu.Item>
+              )}
+              {currentUser.hasPermission('create_dashboard') && (
+                <Menu.Item key="new-dashboard">
+                  <a className="clickable" onClick={CreateDashboardDialog.showModal}>New Dashboard</a>
+                </Menu.Item>
+              )}
+              <Menu.Item key="new-alert">
+                <a href="alerts/new">New Alert</a>
+              </Menu.Item>
+            </Menu>
+          )}
+        >
+          <Button type="primary" data-test="CreateButton">
+            Create <Icon type="down" />
+          </Button>
+        </Dropdown>
+      </div>
+      <div className="header-logo">
+        <a href="./">
+          <img src={logoUrl} alt="Redash" />
+        </a>
+      </div>
+      <div>
+        <Input.Search
+          className="searchbar"
+          placeholder="Search queries..."
+          data-test="AppHeaderSearch"
+          onSearch={onSearch}
+        />
+        <Menu mode="horizontal" selectable={false}>
+          <Menu.Item key="help">
+            <HelpTrigger type="HOME" className="menu-item-button" />
+          </Menu.Item>
+          {currentUser.isAdmin && (
+            <Menu.Item key="settings">
+              <a href="data_sources" className="menu-item-button">
+                <i className="fa fa-sliders" />
```

Comment:
Considering the Help Trigger has a Tooltip on its own, wdyt about adding a Tooltip here as well? So will be a "standard" Icon Button behavior

Reply:
Done. Also converted it to `<Button>` so it has the cool click animation.